build.sh

```diff
@@ -135,10 +135,10 @@ if hasArg clean; then
 fi
 
 if (( ${BUILD_ALL_GPU_ARCH} == 0 )); then
-    GPU_ARCH="-DGPU_ARCHS="
+    CUDF_CMAKE_CUDA_ARCHITECTURESS="-DCMAKE_CUDA_ARCHITECTURES="
```

Comment:
Is there a reason it's `CUDF_CMAKE_CUDA_ARCHITECTURESS` instead of `CUDF_CMAKE_CUDA_ARCHITECTURES`?

Reply:
"What does it have in its pocketses, precious?"

Reply:
lol woops!